files/justfiles/secureblue.just

```diff
@@ -80,17 +80,19 @@ remove-kargs-hardening:
     echo "Hardening kargs removed."
 
 # Harden flatpaks by preloading hardened_malloc (highest supported hwcap)
-harden-flatpak:
+harden-flatpak ACTION="":
```

Comment:
could be named FLATPAK instead?

also, should be noted in the comment

Reply:
done